README.md

```diff
@@ -47,6 +47,8 @@ it is converted to an attribute set equivalent to `{ root = theArg; }`.
 | `version` | The version of the derivation. |
 | `src` | Used by `naersk` as source input to the derivation. When `root` is not set, `src` is also used to discover the `Cargo.toml` and `Cargo.lock`. |
 | `root` | Used by `naersk` to read the `Cargo.toml` and `Cargo.lock` files. May be different from `src`. When `src` is not set, `root` is (indirectly) used as `src`. |
+| `allRefs` | Whether to fetch all refs while fetching Git dependencies. Useful if the wanted revision isn't in the default branch. |
+| `gitSubmodules` | Whether to fetch submodules while fetching Git dependencies. |
```

Comment:
If this maps to `builtins.fetchGit`'s submodule support, we should note here that this only works for Nix 2.4+.